src/views/Detail.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-container>
+    <v-row dense>
+      <v-col cols="12">
+        <AppDetail v-bind:detailed="detailed" />
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import AppDetail from '@/components/app-detail/app-detail.vue';
+import api from '@/api';
+
+export default {
+  name: 'Detail',
+  components: {
+    AppDetail,
+  },
+  data() {
+    return {
+      detailed: {},
```

Comment:
I recommend rename **detailed** to **trackDetail**

Reply:
done